docs/_static/css/dark.css

```diff
@@ -46,7 +45,9 @@ em.property {
     background-color: #5a5a5a;
 }
 
-
+.rst-content pre{
+    background: none;
+}
```

Comment:
One more thing, I'd change the background here to white and a add thicker border here instead (https://github.com/ethereum/solidity/issues/11900#issuecomment-919297759, https://github.com/ethereum/solidity/issues/11900#issuecomment-919311493).

Reply:
@axic What do you think? I think that green ones looked slightly better but white ones are not bad either so I'm fine with them.